ocs_ci/ocs/ocp.py

```diff
@@ -307,10 +309,15 @@ def wait_for_resource(
                 in_condition = []
                 sample = sample['items']
                 for item in sample:
-                    if self.get_resource_status(
-                        item.get('metadata').get('name')
-                    ) == condition:
-                        in_condition.append(item)
+                    try:
+                        _name = item.get('metadata').get('name')
+                        if self.get_resource_status(_name)  == condition:
+                            in_condition.append(item)
+                    except CommandFailed as ex:
```

Comment:
this will change the behavior.
There might be calls to this that expect it to throw an exception in case you can't find that resource

Reply:
Forgot to send the review @RazTamir, but this is the comment I was talking on the mtg.